src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/AvoidUnusedPrivateFieldsTests.cs

```diff
@@ -348,29 +345,27 @@ public async Task CA1823_VisualBasic_StructLayoutAttribute_LayoutKindExplicit_Di
 Public Class [Class]
     Private field As Integer
 End Class
-", CompilerDiagnostics.None,
```

Comment:
💭 Interesting, I guess Visual Basic doesn't have the equivalent of CS0625

/cc @dotnet/roslyn-compiler